core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -43,14 +51,38 @@
 
   private final String mRoot;
   private final AtomicInteger mSize = new AtomicInteger(0);
+  private final long mPageSize;
+  private final Pattern mPagePattern;
 
   /**
    * Creates a new instance of {@link LocalPageStore}.
    *
    * @param options options for the local page store
+   * @throws IOException when fails to create a {@link LocalPageStore}
    */
-  public LocalPageStore(LocalPageStoreOptions options) {
+  public LocalPageStore(LocalPageStoreOptions options) throws IOException {
     mRoot = options.getRootDir();
+    mPageSize = options.getPageSize();
+    Path rootDir = Paths.get(mRoot);
+    mPagePattern = Pattern.compile(
+        String.format("%s/%d/(\\d+)/(\\d+)", rootDir.toString(), mPageSize));
```

Comment:
`Pattern.quote(rootDir.toString())` to escape the root path?

Reply:
Good catch! Updated.